### PR TITLE
[DO NOT MERGE] feat(docs): march milestone

### PR DESCRIPTION
**Pull request by dhmlau**
Overall theme: finish as much migration guide as possible. 

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/_mar-milestone_.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  - [ ] Spike on migrating Angular SDK client (1.x and 2+) (to be created -
+        originated from https://github.com/strongloop/loopback-next/issues/3954)
+
+- [ ] Spike: robust handling of ObjectID type for MongoDB #3456
+
+## Looking for community contributions
+
+_Community contributors: Want to contribute but don't know where to start? Here
+is our wish list for this month:_
```
Comment (dhmlau) [inline docs/_mar-milestone_.md]: Model.toObject() should preserve prototypes (e.g. Date and ObjectID values) #3607

Comment (dhmlau) [reply docs/_mar-milestone_.md]: Looking from the list of "good first issue" with highest number of upvotes: https://github.com/strongloop/loopback-next/issues?page=1&q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+sort%3Areactions-%2B1-desc.

- Add support for "include" and "fields" to findById (REST API) https://github.com/strongloop/loopback-next/issues/1721
- Best practice for encrypting model properties https://github.com/strongloop/loopback-next/issues/2095 . _Not sure if this is only good for a very limited set of use cases._
- Complex OpenAPI Validations with `@property` https://github.com/strongloop/loopback-next/issues/1624

File: docs/_mar-milestone_.md [hunk 2]
```diff
@@ -45,3 +52,4 @@ is our wish list for this month:_
       values) #3607
 - [ ] Add support for "include" and "fields" to findById (REST API) #1721
```
Comment (raymondfeng) [inline docs/_mar-milestone_.md]: This is already supported. I added a test for `fields` as part of https://github.com/strongloop/loopback-next/pull/4745.

File: docs/_mar-milestone_.md [hunk 3]
```diff
@@ -45,3 +52,4 @@ is our wish list for this month:_
...
 - [ ] Complex OpenAPI Validations with @property #1624
+- [ ] Sugar decorators for Filter and Where parameters #1749
```
Comment (raymondfeng) [inline docs/_mar-milestone_.md]: Implemented by https://github.com/strongloop/loopback-next/pull/4745

File: docs/_mar-milestone_.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        based authentication to LB4 #4521
+  - [ ] [8] How to migrate apps using loopback-component-passport #3958
+  - [ ] [5][migration/auth] Describe how to simulate ACLs with LB4 authorization
+        #4522
+
+- [ ] Miscellaneous
+  - [ ] Belongs-To relation tries to resolve for foreign key NULL #4372
+  - [ ] How to build models, repositories and controllers dynamically at runtime
+        #4296
+
```
Comment (bajtos) [inline docs/_mar-milestone_.md]: Let's add the story to update LB3-related docs to accommodate the transition to Maintenance LTS, see https://github.com/strongloop/loopback/issues/4306.

File: docs/_mar-milestone_.md (hunk 1 above)
Comment (bajtos) [reply docs/_mar-milestone_.md]: The list proposed above looks good to me 👍

**Comment by agnes512** [comment]
adding 2 bug fix:
https://github.com/strongloop/loopback-next/issues/4751
https://github.com/strongloop/loopback-next/issues/4744

**Comment by dhmlau** [comment]
Thanks for everyone's review! I think the March milestone is good to go.  We'll commit 37 points. 
@agnes512, could you please create a GH issue for that? Thanks. We can use it in the Feb blog.

**Comment by dhmlau** [comment]
Closing this PR as https://github.com/strongloop/loopback-next/issues/4767 is opened.

**Pull request closed without merging**